tools/analyzer_plugin/playground/analysis_options.yaml

```diff
@@ -3,3 +3,8 @@ include: package:workiva_analysis_options/v1.yaml
 analyzer:
   plugins:
     over_react
+
+over_react:
+  errors:
+    # Opt into this diagnostic that's disabled by default.
+    over_react_annotation_required_prop: info
```

Comment:
Is `info` the default severity of this?

Reply:
It doesn't really have a default severity since it's disabled by default, and to enable it you also have to specify a severity